drinfomon.lic

```diff
@@ -704,7 +704,6 @@ end
 
 put('exp all')
 pause 1
-put('info') unless dead?
```

Comment:
Leaving this in caused a double send of 'info'. It seems to take a few seconds for infomon to parse and register the information displayed, causing the conditional on line 854 to be true